apps/src/code-studio/initApp/loadApp.js

```diff
@@ -66,10 +67,13 @@ export function setupApp(appOptions) {
         appOptions.app === 'weblab'
       ) {
         $('#clear-puzzle-header').hide();
-        // Only show Version History button if the user owns this project
-        if (project.isEditable()) {
-          $('#versions-header').show();
-        }
+      }
+      // If user is project owner, or teacher viewing student work
```

Comment:
(very minor) This comment feels like it's missing a few words (e.g. "Only show Version History button"...)?

Reply:
Good call- updated:)